android/app/src/sharedTest/java/com/microsoft/codepush/common/testutils/CommonFileTestUtils.java

```diff
@@ -30,13 +31,39 @@ public static File getRealTestFolder() {
      * @return real test file.
      */
     public static File getRealFile() throws IOException {
+        return getRealNamedFile("file.txt");
+    }
+
+    /**
+     * Creates a real (not mocked) file for testing.
+     *
+     * @param fileName name for the file.
+     * @return real test file.
+     */
+    public static File getRealNamedFile(String fileName) throws IOException {
         File testFolder = getRealTestFolder();
         testFolder.mkdirs();
-        File realFile = new File(testFolder, "file.txt");
+        File realFile = new File(testFolder, fileName);
         realFile.createNewFile();
         return realFile;
     }
 
+    /**
+     * Creates a real (not mocked) file for testing.
+     *
+     * @param fileName name for the file.
+     * @param content  content of file.
+     * @return real test file.
+     */
+    public static File getRealNamedFileWithContent(String fileName, String content) throws IOException {
+        File file = getRealNamedFile(fileName);
+        FileWriter fw = new FileWriter(file);
```

Comment:
Please get rid of the variable names like `fw` - better name it `fileWriter`.

Reply:
@annakocheshkova got it.